docs/static/breaking-changes.asciidoc

```diff
@@ -32,6 +32,13 @@ Here are the breaking changes for 8.0.
 [float]
 ==== Changes in Logstash Core
 
+[float]
+[[ruby-execution-engine]]
+===== Ruby Execution Engine removed
+The Execution Engine transforms pipeline definitions into executable code, and manages that code's execution.
+Since {ls} 7.0, the Java Execution Engine (JEE) has been the default, but users could still configure their installation to use the legacy Ruby Execution Engine (REE).
+The option to use REE is removed in 8.0.
+
```

Comment:
@yaauie, looks like we added this issue in https://github.com/elastic/logstash/pull/13693:

>[[bc-ruby-engine]]
>===== Ruby Execution Engine removed
>The Java Execution Engine has been the default engine since Logstash 7.0, and works with plugins written in either Ruby or Java.
>Removal of the Ruby Execution Engine will not affect the ability to run existing pipelines. https://github.com/elastic/logstash/pull/12517[#12517]

Unless you want to tweak what's there, I think we can close this one.